src/scifem/interpolation.py

```diff
@@ -55,37 +55,46 @@ def prepare_interpolation_data(
     array_evaluated = compiled_expr.eval(mesh, np.arange(num_cells, dtype=np.int32))
     assert np.prod(Q.value_shape) == np.prod(expr.ufl_shape)
 
-    im = Q.element.basix_element.interpolation_matrix
-
     # Get data as (num_cells*num_points,1, expr_shape, num_test_basis_functions*test_block_size)
     expr_size = int(np.prod(expr.ufl_shape))
     array_evaluated = array_evaluated.reshape(
         num_cells * q_points.shape[0], 1, expr_size, V.dofmap.bs * V.dofmap.dof_layout.num_dofs
     )
-    jacobian = dolfinx.fem.Expression(ufl.Jacobian(mesh), q_points)
-    detJ = dolfinx.fem.Expression(ufl.JacobianDeterminant(mesh), q_points)
-    K = dolfinx.fem.Expression(ufl.JacobianInverse(mesh), q_points)
-    jacs = jacobian.eval(mesh, np.arange(num_cells, dtype=np.int32)).reshape(
-        num_cells * num_points, mesh.geometry.dim, mesh.topology.dim
-    )
-    detJs = detJ.eval(mesh, np.arange(num_cells, dtype=np.int32)).flatten()
-    Ks = K.eval(mesh, np.arange(num_cells, dtype=np.int32)).reshape(
-        num_cells * num_points, mesh.geometry.dim, mesh.topology.dim
-    )
 
-    Q_vs = Q.element.basix_element.value_size
+    try:
+        Q_vs = Q.element.basix_element.value_size
+    except RuntimeError:
+        Q_vs = 1  # If we do not have a basix element, assume value size is 1
```

Comment:
Is `RuntimeError` correct here? Shouldn't it be `AttributeError`? Also why does this happen in the first place? Perhaps we should emit some warning?